pages/game.js

```diff
@@ -56,6 +61,7 @@ export default class Index extends Component {
             const newState = {
                 currentGrid: newCurrentGrid,
                 isVictory,
+                suggest: 0,
```

Comment:
What's this ? Isn't it supposed to be `suggestedTile` ?

Reply:
It's a way to reset the advice after a move. But you are right it's now suggestedTile.